abstrak/abstrak-en.tex

```diff
@@ -7,21 +7,21 @@
 \vspace{2ex}
 
 \begingroup
-  % Menghilangkan padding
-  \setlength{\tabcolsep}{0pt}
+% Menghilangkan padding
+\setlength{\tabcolsep}{0pt}
 
-  \noindent
-  \begin{tabularx}{\textwidth}{l >{\centering}m{3em} X}
-    % Ubah kalimat berikut dengan nama mahasiswa
-    \emph{Name}     &:& Elon Reeve Musk \\
+\noindent
+\begin{tabularx}{\textwidth}{l >{\centering}m{3em} X}
+  % Ubah kalimat berikut dengan nama mahasiswa
+  \emph{Name}     & : & \name{}                                                             \\
 
-    % Ubah kalimat berikut dengan judul tugas akhir dalam Bahasa Inggris
-    \emph{Title}    &:& \emph{Anti-Gravity Based Energy Calculation on Outer Space Rockets} \\
+  % Ubah kalimat berikut dengan judul tugas akhir dalam Bahasa Inggris
+  \emph{Title}    & : & \engtatitle{} \\
 
-    % Ubah kalimat-kalimat berikut dengan nama-nama dosen pembimbing
-    \emph{Advisors} &:& 1. Nikola Tesla, S.T., M.T. \\
-                    & & 2. Wernher von Braun, S.T., M.T. \\
-  \end{tabularx}
+  % Ubah kalimat-kalimat berikut dengan nama-nama dosen pembimbing
+  \emph{Advisors} & : & 1. \advisor{}                                                       \\
+                  &   & 2. \coadvisor{}                                                     \\
```

Comment:
Since we're using variables, i don't think these comments are needed. Check the other comments too.